src/sage/schemes/elliptic_curves/ell_number_field.py

```diff
@@ -1,21 +1,25 @@
 # -*- coding: utf-8 -*-
-r"""
-Elliptic curves over number fields
+r"""Elliptic curves over number fields
```

Comment:
I would revert this as our convention is one-line on a new line from the `"""`.

Reply:
Will do.  My emacs python mode always does this, I wish it would not!